src/logging_config.py

```
@@ -0,0 +1,116 @@
+"""Structured JSON logging configuration.
+
+This module configures structured logging for the application,
+outputting logs in JSON format suitable for log aggregation systems.
+"""
+
+import logging
+import sys
+from typing import Any
+
+from pythonjsonlogger import jsonlogger
```

Pylint (4.0.4)

[error] Unable to import 'pythonjsonlogger'

(E0401)